ClientApp/viking-vault/src/UserDashboard/CreateAutomaticPaymentModal.tsx

```diff
@@ -2,14 +2,14 @@ import React from "react";
 import { constants } from "../Resources/Constants.js";
 import '../AdminDashboard/AttachCardModal.css';
 
-const regexCheckIfPositiveFloat = "^[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?$";
+const regexCheckIfPositiveFloat = "^[-+]?[0-9]*\.?[0-9]+([-+]?[0-9]+)?$";
```

Comment:
Please extract this and move into the constants file as it seems like the same regex is used in multiple pages within the app. So when you want to make a change in this regex you then have to modify it in all areas, versus if you were to extract it you would only have to modify it in 1 place